client/src/containers/ProfileBoxTester.js

```diff
@@ -36,6 +37,10 @@ export const ProfileBoxTester = () => {
           "Pre-professional",
         ]}
       />
+      <FrequencyTag
+        frequency="3"
+        weekly="true"
```

Comment:
instead of passing in true/false values as a string, passing them in as {true} will make this easier to read/more bug-free